lib/graphql_pagination/collection_type.rb

```diff
@@ -14,8 +14,8 @@ def collection_type(
 
         Class.new(collection_base) do
           graphql_name type_name
-          field :collection, [source_type], null: false
-          field :metadata, metadata_type, null: false
+          field :collection, [source_type], null: false, describe: "A collection of paginated #{graphql_name}"
+          field :metadata, metadata_type, null: false, describe: "Pagination Metadata for navigating the Pagination"
```

Comment:
raises:

```
ArgumentError:
  unknown keyword: :describe
```

Reply:
Sorry about that, i updated it to be `description`